testing/adios2/install/C/main.c

```diff
@@ -0,0 +1,11 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ */
+
+#include <adios2_c.h>
+
+int main(void)
+{
+  return 0;
```

Comment:
Let's go ahead and add a symbol to validate linking.  It doesn't need to even be working code, so long as it compiles, links, and doesn't segfault.
```C
int main(void)
{
  adios2_finalize(NULL);
  return 0;
}
```
@williamfgc would this snippet execute without segfaulting?